src/Symfony/Component/Console/Input/InputInterface.php

```diff
@@ -18,6 +18,9 @@
  * InputInterface is the interface implemented by all input classes.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @method string __toString() Returns a stringified representation of the args passed to the command.
```

Comment:
AFAIK if someone does not implement this method, a deprecation is thrown.

I would like to have a test case which asserts this deprecation (a missing implementation) and has the "@group legacy".

Not sure I missed sth.

Reply:
I'll see how to achieved that. Thanks for clarification. 
The unit test to verify the deprecation should be part of the docs repo?